src/components/users/registration/RegistrationPage.js

```diff
@@ -12,7 +12,7 @@ const RegistrationPage = () => {
   const registerUser = (event) => {
     event.preventDefault()
     client.post('/user', user, false)
-      .then(res => setRegisterResponse(res.data))
+      .then(res => setRegisterResponse(res.data), console.log("hi", registerResponse))
```

Comment:
Do we need this console.log now?

If not, it's always best to remove them so we can keep the console clean.

Reply:
My bad. Didn't realise that was still there🤦